public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchMetricsQueryRunner.ts

```diff
@@ -14,38 +13,18 @@ import {
   ScopedVars,
 } from '@grafana/data';
 import { TemplateSrv } from '@grafana/runtime';
-import { notifyApp } from 'app/core/actions';
-import { createErrorNotification } from 'app/core/copy/appNotification';
-import { store } from 'app/store/store';
-import { AppNotificationTimeout } from 'app/types';
 
-import { ThrottlingErrorMessage } from '../components/Errors/ThrottlingErrorMessage';
-import memoizedDebounce from '../memoizedDebounce';
 import { migrateMetricQuery } from '../migrations/metricQueryMigrations';
 import { CloudWatchJsonData, CloudWatchMetricsQuery, CloudWatchQuery } from '../types';
 import { filterMetricsQuery } from '../utils/utils';
 
 import { CloudWatchRequest } from './CloudWatchRequest';
 
-const displayAlert = (datasourceName: string, region: string) =>
-  store.dispatch(
-    notifyApp(
-      createErrorNotification(
-        `CloudWatch request limit reached in ${region} for data source ${datasourceName}`,
-        '',
-        undefined,
-        React.createElement(ThrottlingErrorMessage, { region }, null)
-      )
-    )
-  );
+const getThrottlingErrorMessage = (region: string, message: string) =>
+  `Please visit the AWS Service Quotas console at https://${region}.console.aws.amazon.com/servicequotas/home?region=${region}#!/services/monitoring/quotas/L-5E141212 to request a quota increase or see our documentation at https://grafana.com/docs/grafana/latest/datasources/cloudwatch/#manage-service-quotas to learn more. ${message}`;
```

Comment:
Is it possible for these to be links, or is that just not how errors work?

Reply:
The `DataQueryError` interface only accepts strings for messages.